server/server.js

```diff
@@ -0,0 +1,41 @@
+/* eslint no-console:0 */
+var loopback = require('loopback');
+var boot = require('loopback-boot');
+var ServiceManager = require('./service-manager');
```

Comment:
I'm commenting on the complete file diff... (actually, I'm reading in vim, diffs suck for something this large, and then looking for files in the complete PR diff).

All of which is to say: did you not push that change?